### PR TITLE
CNV VCFs

### PR DESCRIPTION
**Pull request by nvnieuwk**
This PR contains:

- Creation of CNV VCFs
- Merge CNV VCFs with Jasmine (when more than one caller is used)
- Annotation of CNV VCFs with the SV annotation workflow
- Some refactoring of the SV code to make code reusable for CNVs
- Small fixes to the tests (proper tests will be created in another PR)

### CODE REVIEW

File: conf/modules.config
[the remark this answers is not in the document]
Comment (nvnieuwk) [reply conf/modules.config:1]: Yep, basically. Jasmine shouldn't run on only one VCF :). the changes done here are mainly to make it easier for myself in the future to add more types of output

File: subworkflows/local/vcf_merge_jasmine/main.nf
Comment (mvheetve) [inline subworkflows/local/vcf_merge_jasmine/main.nf:1]: Not entirely sure what goes on here. If you use more than 1 caller, then Jasmine gets a reheader to make sure there is consensus across the different vcfs?

Comment (nvnieuwk) [reply subworkflows/local/vcf_merge_jasmine/main.nf:1]: Yes, Jasmine doesn't output the headers of the VCFs so I have to reheader them to make further tools work

**Review by mvheetve — APPROVED** [review]
Approved!

**Pull request merged**